### PR TITLE
Finished: Implement Calculator UI with components.

### PR DESCRIPTION
**Pull request by Zafron047**
Dear Reviewer,

This pull request only introduces the UI of calculator. I have implemented the below **changes:**

- [x] Created a new directory called "components"
- [x] Added a new file called "Calculator.js" inside the "components" directory
- [x] Implemented the design of the calculator using JSX and CSS
- [x] Utilized functional components for improved performance and readability
- [x] Created at least one child component to enhance modularity

Please review the code changes and provide your valuable feedback. Your contributions are highly appreciated in maintaining the quality and reliability of our codebase.

Thank you for your attention and support! 😺 

Best regards,

@Zafron047

### CODE REVIEW

File: src/components/UI.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  background-color: #ffc266;
+}
+
+button:hover {
+  background-color: #d9d9d9;
+}
+
+button:active {
+  background-color: #bfbfbf;
+}
```
Comment (Ibizugbe) [inline src/components/UI.css]: Nice job so far👍. Your calculator design looks really beautiful😉. But for the sake of meeting the requirements of this project, let's try to stick to the given design😉. when we are done with the entire project you can further add styles to your taste to make it look better😉. But for now, let's stick with the design🙏.

image


your design
image

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Zafron047 👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are properly setup and they are passing👍.
- [x] A component folder has been created with a `calculator.js` component👍.
- [x] The `calculator.js` has a child component👍.
- [x] Gitflow is used👍.


## Required Changes ♻️

- [x] Just as required for this milestone, kindly take away the boilerplate styles in your `index.css` file😉.
image


image



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/UI.css (hunk 1 above)
Comment (Zafron047) [reply src/components/UI.css]: @Ibizugbe Thanks a lot for your kind review. 

- [x] I have removed the index.css file.
- [x] I have tried as best to restore styling as the original design.

Best,

@Ibizugbe

**Review by Ibizugbe — APPROVED** [review]
Hi @Zafron047 👋👋, 

StandingOvationGIF (8)


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅

- [x] Linters are passing👍.
- [x] Boilerplate styles from `index.css` has been removed👍.
- [x] Project has been styled to look like the design👍.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Zafron047** [comment]
@Ibizugbe Thanks a lot for your kind time reviewing it. 😄 

Best,

@Zafron047

**Pull request merged**